src/main/java/bio/terra/service/resourcemanagement/google/GoogleBucketService.java

```diff
@@ -276,6 +281,7 @@ public void updateBucketMetadata(String bucketName, String flightId) {
      * @return a reference to the bucket as a GCS Bucket object
      */
     private Bucket newCloudBucket(GoogleBucketResource bucketResource) {
+        boolean doVersioning = Arrays.stream(env.getActiveProfiles()).noneMatch(env -> env.contains("test"));
```

Comment:
does this stream need to be collected?

Reply:
I think you only need to `collect` when you need to mutate and reduce the stream into something else.